src/common/transformations/tests/common_optimizations/gru_cell_fusion.cpp

```diff
@@ -21,7 +21,7 @@ namespace {
 
 enum class WeightsFormat { zr, rz };
 
-Output<Node> create_activation_by_name(const string& activation_name, const Output<Node>& input) {
+Output<Node> create_activation_by_name(const std::string& activation_name, const Output<Node>& input) {
```

Comment:
This is a c++ file, so the scope is limited by this file, then why doesn't https://github.com/openvinotoolkit/openvino/pull/21045/files#diff-eae5a6bb6749fcebc04ed355c845d1de796c588987df08133c99d256203f8ea1R175 work for you as in the original PR?